tests/conftest.py

```diff
@@ -71,8 +71,15 @@ def request_cart_with_item(product_in_stock, request_cart):
 
 
 @pytest.fixture
-def order(billing_address):
-    return Order.objects.create(billing_address=billing_address)
+def order(billing_address, customer_user):
+    return Order.objects.create(
+        billing_address=billing_address,
+        user_email=customer_user.email,
+        total_net=Money(100, currency='USD'),
```

Comment:
Lets drop `currency=` form `Money()`, its remnant from back when we had `Price()` where it was third arg and we already dropped that in Saleor's codebase.